### DISCUSSION TITLE
How to start using semantic release from 0.0.0 version instead of 1.0.0 ?

### DISCUSSION THREAD

**Discussion by vitalik203** [semantic-release/semantic-release#3240] [Configuration]
How to start using semantic release from 0.0.0 version instead of 1.0.0 ?

**Comment by travi** [comment]
https://semantic-release.gitbook.io/semantic-release/support/faq#can-i-set-the-initial-release-version-of-my-package-to-0.0.1

**Reply by vitalik203** [reply]
Can I change a parameter, that alows us to start versioning from 0.0.0, or probably you know anything else to help me with it ?
